src/main/java/com/uber/cadence/workflow/WorkflowUtils.java

```diff
@@ -0,0 +1,43 @@
+/*
+ *  Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ *  use this file except in compliance with the License. A copy of the License is
+ *  located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package com.uber.cadence.workflow;
+
+import com.cronutils.utils.StringUtils;
+import com.uber.cadence.SearchAttributes;
+import com.uber.cadence.converter.DataConverter;
+import com.uber.cadence.converter.JsonDataConverter;
+import java.nio.ByteBuffer;
+
+public class WorkflowUtils {
+  private static final DataConverter jsonConverter = JsonDataConverter.getInstance();
+
+  public static <T> T getValueFromSearchAttributes(
```

Comment:
IMO this should not be part of the library. Cadence library should only return byte array. User is responsible to translate that back to strongly typed object, with whatever data converter they prefer.